ci/pipeline-dev.yml

```diff
@@ -343,13 +344,19 @@ jobs:
           <<: *env-cf
           APP_ENV: ((deploy-env))
           CF_APP_NAME: pages-((deploy-env))
+      - task: create-test-users
+        file: src/ci/partials/create-test-users.yml
+        image: node
+        params:
+          DOMAIN: ((((deploy-env))-pages-domain))
+          APP_ENV: ((deploy-env))
+          PRODUCT: pages
```

Comment:
For another issue but we should probably remove the `PRODUCT` env from the app and deployment.